src/components/DetailsModal.tsx

```diff
@@ -72,44 +90,153 @@ function DetailsModal({
     }
   }, [auth, submission, isModalOpen]);
 
-  const getErrorMessages = (): string[] => {
-    if (!submission.issues?.length) return [];
-
-    return submission.issues;
+  const getHeaderContent = () => {
+    return (
+      <>
+        <div className="grid-row flex-row flex-align-center padding-bottom-2">
+          <Icons.PaperLines />
+          <h2 className="font-sans-md text-bold padding-left-1">
+            {details.filename}
+          </h2>
+        </div>
+        <span className="padding-bottom-2">
+          <Pill
+            variation="info"
+            altText={displayValues.label}
+            label={displayValues.label}
+            color={displayValues.pillColor}
+            icon={displayValues.pillIcon}
+          />
+        </span>
+        <span className="font-sans-md padding-bottom-2">
+          Stage: {details.lastService} - {details.lastAction}
+        </span>
+      </>
+    );
   };
 
-  const getContent = () => {
-    const messages = getErrorMessages();
-    if (messages.length > 0) {
-      return (
-        <Alert heading="Failed" type="error">
-          {messages.length} Error(s) were detected
-          {messages.map((issue: string) => (
-            <p
-              key={issue}
-              className="border-1px radius-md border-secondary-light margin-y-2 padding-105">
-              {issue}
-            </p>
-          ))}
-        </Alert>
-      );
-    }
+  const getFailedContent = () => {
+    if (details.status.toLowerCase() != "failed") return null;
 
-    if (submission.status.toLowerCase().includes("complete")) {
-      return (
-        <Alert heading="Complete" type="success">
-          File submission is complete
-        </Alert>
-      );
-    }
+    const issues = details.reports[0]?.issues;
 
-    if (submission.status.toLowerCase().includes("uploading")) {
+    if (!issues.length)
       return (
-        <Alert heading="Processing" type="info">
-          File submission is processing
+        <Alert
+          className="margin-y-1"
+          heading={`Error in stage ${details.lastService} - ${details.lastAction}`}
+          type="error">
+          No specifics were given, please see full attached report below.
         </Alert>
       );
-    }
+
+    return (
+      <Alert
+        className="margin-y-1"
+        heading={`${issues.length} error(s) returned in stage ${details.lastService} - ${details.lastAction}`}
+        type="error">
+        Please review and address these errors.
```

Comment:
UX: the end user doesn't really have a way to address these issues within the portal, so do we really want to say this?

Reply:
This is an Abbey question and she signed off on this. Plus I know for a fact that this details modal is going to get changed a bunch more times before G.A.